components/Buttons/LikeButton.vue

```diff
@@ -36,12 +36,12 @@ export default Vue.extend({
     }
   },
   mounted() {
-    this.$store.subscribe((mutation) => {
+    this.$store.subscribe((mutation, state) => {
```

Comment:
Imo we shouldn't listen to the websocket messages here, we are already listening to them in the `card` component, which should update this via props when there is a change

Reply:
We can rework this on #639. The like button is not always used in card components, but also in pages and such.